src/syrupy/extensions/amber.py

```diff
@@ -188,6 +188,18 @@ def serialize_iterable(
             + cls.with_indent(close_paren, depth)
         )
 
+    @classmethod
+    def serialize_callable(
+        cls,
+        data: "SerializableData",
+        *,
+        depth: int = 0,
+        visited: Optional[Set[Any]] = None,
+    ) -> str:
+        return cls.with_indent(
+            f"{cls.object_type(data)} ({cls.MarkerDepthMax()})", depth
+        )
```

Comment:
There are some options here. Should include the function name? Doc string? type information if available?

What's a use case where someone would snapshot a function?

Reply:
We can definitely get the type information, but I was wondering if it's better to just ignore methods when in a class. That way we won't need to account for callable.

Reply:
Personally I'm in favour of ignoring and leaving this up to a community built extension of needed, or open issues.